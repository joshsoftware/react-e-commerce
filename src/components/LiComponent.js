import React from 'react';
import PropTypes from 'prop-types';

const LiComponent = ({ data }) => {
  return <li className="rounded-sm">{data}</li>;
};

export default LiComponent;

LiComponent.propTypes = {
  data: PropTypes.oneOfType([PropTypes.element, PropTypes.string])
};
