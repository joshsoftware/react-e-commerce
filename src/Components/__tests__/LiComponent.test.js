import React from 'react';
import { render } from '@testing-library/react';
import LiComponent from '../LiComponent';

describe('ListComponent ', () => {
  it('Must return entire Listcontainer', () => {
    const { asFragment } = render(<LiComponent />);

    expect(asFragment()).toMatchSnapshot();
  });
});
