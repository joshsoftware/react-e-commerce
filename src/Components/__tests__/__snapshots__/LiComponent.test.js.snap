// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListComponent  Must return entire Listcontainer 1`] = `
<DocumentFragment>
  <li
    class="rounded-sm"
  />
</DocumentFragment>
`;
